### ISSUE TITLE
Headless progress

### ISSUE THREAD

**Issue by code-bunny** [maglevhq/maglev-core#67]
Forgive if I am asking in the wrong place, I see in the roadmap you are working on a headless version that would support NextJS. Is there an expected completion time before we go investing in building a full stack Rails site?

**Comment by did** [comment]
@code-bunny you're right, you should have asked in the [Slack channel ](https://join.slack.com/t/maglevhq/shared_invite/zt-1hy671m34-CSfumgCACIeI7veuiCWcig) instead but that's okay :-)

So I haven't worked on it for while now. It's a pretty complex task and I guess it was around 60% done. Besides, I targeted NuxtJS first (because I used to write VueJS apps) although the JS "foundation" was framework agnostic. 

I've no idea when I will work on it again. Unless the community pushes in that direction or if a client sponsors that feature...

Sorry :-(

**Comment by code-bunny** [comment]
We heavily use Strapi at Opera and in my client projects in Code Bunnies. But I very much prefer to keep in the Ruby system and your CMS is quite lovely. Will continue to build up some projects for now and see how much work it'll be to make it headless internally then push that back to the main project.
